README.md

```diff
@@ -1,36 +1,70 @@
 tendrl-ansible
 ==============
 
-Ansible roles and playbooks for [Tendrl](http://tendrl.org/)!
+[Ansible](https://docs.ansible.com/ansible/latest/index.html) roles and
+playbooks for [Tendrl](http://tendrl.org/)!
 
-Clone me:
 
-```bash
-git clone https://github.com/Tendrl/tendrl-ansible.git
+## What does it do?
+
+tendrl-ansible automates **installation of Tendrl** based description from
+[Tendrl wiki](https://github.com/Tendrl/documentation/wiki). You
+should check [installation documentation
+there](https://github.com/Tendrl/documentation/wiki/Tendrl-release-latest) to
+have basic understanding of various machine roles in Tendrl cluster before
+using tendrl-ansible.
+
+
+## How to get tendrl-ansible?
+
+Just clone this repo:
+
+```
+$ git clone https://github.com/Tendrl/tendrl-ansible.git
 ```
 
+or you can install rpm package from [copr repo `tendrl/release` with stable
+Tendlr upstream
+builds](https://copr.fedorainfracloud.org/coprs/tendrl/release/):
 
-**NOTE:** The `master` branch was completely overwritten on 13th June 2017. The original master branch is now available as the branch `archive`.
+```
+# yum copr enable tendrl/release
+# yum install tendrl-ansible
+```
 
+See [how to enable copr
+repository](https://docs.pagure.org/copr.copr/how_to_enable_repo.html#how-to-enable-repo)
+if you need more help with this step.
 
-## What does it do?
+Note that installing **tendrl-ansible from rpm package is highly recommended
+when you use stable builds** from `tendrl/release` copr. Otherwise just cloning
+the repo is good enough.
+
+
+## Which version of ansible do I need?
+
+Ansible >= 2.4 is required to use tendrl-ansible.
+
+
+## What roles and playbooks are there?
+
+This is a brief overview only, **there is a README file for each role**, where
+you can see more details about each role, along with list of ansible variables
+one can use to tweak it.
+
+Ansible roles for Tendrl:
 
-There are ansible roles for installation of [Tendrl](http://tendrl.org/), based
-on [upstream documentation](https://github.com/Tendrl/documentation/wiki/Tendrl-Package-Installation-Reference):
+* `tendrl-copr`: installs yum repositories with builds provided by Tendrl
+  project, it uses stable `tendrl/release` copr by default
+* `tendrl-server`: installation of **Tendrl Server** machine (where Tendrl api,
+  web and etcd are running)
+* `tendrl-storage-node`: installation of **Tendrl Storage Node** machines
+  (Gluster servers, which you would like to monitor by Tendrl)
```

Comment:
When installed form *rpm*, the roles contain prefix `tendrl-ansible.`, is it ok to reference them here in README file without the prefix? (If I didn't miss something, this README is not updated during package creation.)

Reply:
For the entry about "tendrl-storage-node: installation of Tendrl Storage Node machines (Gluster servers, which you would like to monitor by Tendrl)..."

I might suggest some mention that the tendrl-storage node is the tendrl agent (aka node agent) as it's really an agent, and the "tendrl-storage-node" is may be confusing to the user.

Reply:
@dahorak yeah, good point - I need to unify the role names for both package and source repo